tests/main/selinux-clean/task.yaml

```diff
@@ -45,32 +45,32 @@ execute: |
     session-tool -u test test-snapd-desktop.sh -c 'echo hello world'
     session-tool -u test test-snapd-desktop.sh -c 'mkdir /home/test/foo'
     session-tool -u test test-snapd-desktop.sh -c 'echo foo >/home/test/foo/bar'
-    ausearch -i --checkpoint stamp --start checkpoint -m AVC 2>&1 | MATCH 'no matches'
```

Comment:
We should keep MATCH here as it prints the input it got in case there is no match.